components/stepper/index.css

```diff
@@ -103,10 +103,10 @@
 		--spectrum-numberfield-spacing-min-inline-end-value-to-icon: var(--spectrum-text-to-visual-75);
 		--spectrum-numberfield-spacing-validation-icon-to-stepper: calc(var(--spectrum-number-field-visual-to-in-field-stepper-small) - var(--spectrum-numberfield-button-inline-padding));
 
-		--spectrum-numberfield-button-inline-padding: var(--spectrum-infield-button-inline-edge-to-fill, var(--spectrum-in-field-button-side-edge-to-fill-small));
```

Comment:
We don't have access to the definitions for `--spectrum-infield-button-inline-edge-to-fill` at this spot in the DOM